### ISSUE TITLE
Buttons not not labeled for screen readers

### ISSUE THREAD

**Issue by mgifford** [alohaeditor/Aloha-Editor#592]
There is no way with a screen reader to know what most of the buttons are. In my tests with VoiceOver it doesn't seem like there is a title that is read out.  There seems to be a tooltip, but at looking at the code I don't see a title associated with the button.
